src/AutomatedCar/SystemComponents/CarCollisionDetector.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace AutomatedCar.SystemComponents
+{
+    using AutomatedCar.Models;
+    using AutomatedCar.SystemComponents.Packets;
+    using Avalonia;
+    using Avalonia.Media;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class CarCollisionDetector : SystemComponent
+    {
+        private CollisionPacket packet;
+        private IEnumerable<WorldObject> collidableWorldObjects;
+
+        public CarCollisionDetector(VirtualFunctionBus virtualFunctionBus)
+            : base(virtualFunctionBus)
+        {
+            this.packet = new CollisionPacket();
+            this.collidableWorldObjects = World.Instance.WorldObjects
+                                        .Where(obj => obj.Collideable
```

Comment:
This styling is a bit off.
I would suggest that you either take a break at every `.xy` or you can break only at `.where()` but it shouldn't be indent several tabs. Use 1 tab (4 spaces) to indent.